examples/annotated-skaffold.yaml

```diff
@@ -61,6 +61,15 @@ build:
     # bazel:
     #  target: //:skaffold_example.tar
 
+    # jibMaven is an artifacts built with Maven, typically (but not only) a Java project.
+    # jibMaven:
+    #  module: modulename
+    #  profile: profilename
+
+    # jibGradle is an artifacts built with Gradle, typically (but not only) a Java project.
```

Comment:
And ditto re: `artifacts` or the suggested rewrite.

Reply:
I'll rewrite it